specifyweb/frontend/js_src/lib/components/FormParse/index.ts

```diff
@@ -67,6 +67,7 @@ export type ViewDefinition = {
   readonly viewsetSource: string;
   readonly viewsetId: number | null;
   readonly viewsetFile: string | null;
+  readonly collectionId: number;
```

Comment:
was `collectionid` already part of ViewDefinition but just not included in the type?
otherwise, I don't see the code where you added collectionId to ViewDefinition when fetching from the back-end